activiti-cloud-api-process-model-impl/pom.xml

```diff
@@ -53,5 +53,10 @@
       <artifactId>assertj-core</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
```

Comment:
i am not sure, Would not you get the libraries you need for testing already?

Reply:
changed the whole spring test set for only the required mockito deps

Reply:
we already had junit and assertj, so nothing crazy to add